home/views.py

```diff
@@ -7,3 +9,17 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+def new_question(request):
+    form = QuestionForm
+    if request.method == 'POST':
+        questForm = QuestionForm(request.POST)
+        if questForm.is_valid():
+            questForm = questForm.save(commit=False)
+            for currProfile in Profile.objects.all():
+                if (currProfile.user == request.user):
+                    questForm.profile = currProfile
+                    break
+            questForm.save()
+    return render(request, 'home/questions/new_question.html', {'form': form, 'title': 'New Question'})
```

Comment:
I think after submitting the form you need to go back to Lior's page (question-detail) of the new question.
you can change it after Lior's PR will be merged.

Reply:
You're right, but I thought of making the links in a different PR (with more things). 
Anyway, if Lior's PR will approve before this, I'll do it here.